test/ft/di_errors.cpp

```diff
@@ -1343,10 +1343,3 @@ int main() { di::make_injector<test_config>(); }
     test access_dependency_create = [] {
       expect_compile_fail("", errors(), struct provider{}; int main() { di::bind<int>().create<int>(provider{}); });
     };
-
-#if !defined(__MSVC__) && (__GNUC__ < 6)
-    test access_dependency_is_referable = [] {
```

Comment:
Why this test was removed? We still want it, I think. Maybe we can get rid of MSVC hacks, though?

Reply:
It is not hack. The problem is that dependency_proxy need to be able to call `create` of dependency which is protected...